src/encoded/tests/test_indexing.py

```diff
@@ -20,26 +22,38 @@
 TEST_COLLECTIONS = ['testing_post_put_patch', 'file_processed']
 
 
-@pytest.yield_fixture(scope='session')
-def app(app_settings, use_collections=TEST_COLLECTIONS):
-    """
-    Use to pass kwargs for create_mapping to conftest app
-    """
-    for app in conf_app(app_settings, collections=use_collections, skip_indexing=True):
-        yield app
+@pytest.yield_fixture(scope='session', params=[False])
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [redefinition of unused 'app_settings' from line 16](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4692436)